actors/runtime/indices/indices.go

```diff
@@ -12,66 +10,22 @@ import (
 // it is a passive data structure that allows for convenience access to network indices
 // and pure functions in implementing economic policies given states
 type Indices interface {
-	Epoch() abi.ChainEpoch
-	NetworkKPI() big.Int
-	TotalNetworkSectorWeight() abi.SectorWeight
-	TotalPledgeCollateral() abi.TokenAmount
-	TotalNetworkEffectivePower() abi.StoragePower // power above minimum miner size
-	TotalNetworkPower() abi.StoragePower          // total network power irrespective of meeting minimum miner size
-
-	TotalMinedFIL() abi.TokenAmount
-	TotalUnminedFIL() abi.TokenAmount
-	TotalBurnedFIL() abi.TokenAmount
-	LastEpochReward() abi.TokenAmount
```

Comment:
Blocking: lets keep these accessors for now, at least NetworkKPI, NetworkIndicator, and NetworkTime (changes landed in a separate PR). TotalMined and TotalSupply can be used to compute block reward iteratively. For the other totals, is the idea to compute them in other actors?

Reply:
I was able to remove these because there are *zero callers* of them. When cryptoecon research provides a concrete function that requires something like this, I'll add it back in whatever way makes sense for that value when I know what it is.